tools/pipelines/test-real-service-stress.yml

```diff
@@ -123,3 +123,22 @@ stages:
         env:
           fluid__test__driver__frs: $(automation-fluid-test-driver-frs-stress-test)
           FLUID_TEST_LOGGER_PKG_PATH: ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger # Contains getTestLogger impl to inject
+  # stress tests frs canary
+  - stage:
+    displayName: Stress tests - frs canary
+    dependsOn: []
+    # use a variable group with exclusive lock force only one run at a time and avoid overloading the server/throttling
+    variables:
+    - group: stress-frs-canary
+    jobs:
+    - template: templates/include-test-real-service.yml
+      parameters:
+        poolBuild: Large
+        testPackage: ${{ variables.testPackage }}
+        testWorkspace: ${{ variables.testWorkspace }}
+        artifactBuildId: $(resources.pipeline.client.runID)
+        timeoutInMinutes: 120
+        testCommand: start:frs
```

Comment:
how is this different from the above? will the driver know it is targeting canary and set the endpoint accordingly? this is important as it is necessary differentiation in the logs and alerts

Reply:
When downloading the secrets from the prague-vault, I can see it pulling from canery; https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_apis/build/builds/152245/logs/41

Reply:
How and where is the driver setting the endpoint? 

I think it is here: https://github.com/microsoft/FluidFramework/blob/main/packages/test/test-driver-definitions/src/interfaces.ts#L15 and we should add a similar endpoint as `frs-canary`.

Also, the `start:frs` script is pointing to driverendpoint as `frs`: https://github.com/microsoft/FluidFramework/blob/main/packages/test/test-service-load/package.json#L39

Reply:
change look good to me now. please make sure the automation kusto data looks good too. e.g. that you can see canary runs independently from other frs runs.